plugins/ingest-attachment/src/test/resources/org/elasticsearch/ingest/attachment/test/sample-files/text-in-french.txt

```diff
@@ -0,0 +1 @@
+Allons enfants de la Patrie Le jour de gloire est arrivé. Contre nous de la tyrannie
```

Comment:
It is tested with: https://github.com/elastic/elasticsearch/blob/master/plugins/mapper-attachments/src/test/java/org/elasticsearch/mapper/attachments/LanguageDetectionAttachmentMapperTests.java